ols-2.md

```diff
@@ -17,13 +17,13 @@ photos:
 {% assign all-participants = '' %}
 {% assign all-mentors = '' %}
 {% for project in projects %}
-    {% assign p-pparticipants = '' %}
-    {% for p in project.participants %}
-        {% capture all-participants %}{{ all-participants}}, {{ p }}{% endcapture %}
-    {% endfor %}
-    {% for m in project.mentors %}
-        {% capture all-mentors %}{{ all-mentors }}, {{ m }}{% endcapture %}
-    {% endfor %}
+{% assign p-participants = '' %}
+{% for p in project.participants %}
+{% capture all-participants %}{{ all-participants}}, {{ p }}{% endcapture %}
+{% endfor %}
+{% for m in project.mentors %}
+{% capture all-mentors %}{{ all-mentors }}, {{ m }}{% endcapture %}
+{% endfor %}
```

Comment:
I would be nice to keep the indentation. It helps reading and understanding the large bunch of Jekyll nested code

Reply:
Not intentional. The editor on my pc did that. I'm sorry